cpp/src/parquet/encryption/read_configurations_test.cc

```diff
@@ -237,7 +239,9 @@ TEST_P(TestDecryptionConfiguration, TestDecryption) {
     CheckResults(file_name, decryption_config_num, encryption_config_num);
   }
   // Delete temporary test file.
-  ASSERT_EQ(std::remove(file_name.c_str()), 0);
+  ASSERT_OK_AND_ASSIGN(auto path,
```

Comment:
I don't think this is wrong, I'm just wondering what the reasoning is here for my own education.  Is it to allow paths to be more universal accross platorms?

Reply:
Hmm, I think I added these by accident - I was stumbling around trying to figure out why OpenAsync seemed to have made the encryption tests flaky on Windows. But after reworking it so OpenAsync and Open are separate, that isn't an issue anymore.